src/sandstorm/run-bundle.c++

```diff
@@ -1796,13 +1796,18 @@ private:
       dropPrivs(config.uids);
       clearSignalMask();
 
-      capnp::TwoPartyServer server(kj::heap<BackendImpl>(*io.lowLevelProvider, network));
+      auto redirector = kj::refcounted<CapRedirector>();
+      capnp::Capability::Client coreFactoryCap = kj::addRef(*redirector);
+      TwoPartyServerWithBootstrap server(
+        kj::heap<BackendImpl>(*io.lowLevelProvider, network,
+          coreFactoryCap.castAs<SandstormCoreFactory>()),
+        kj::addRef(*redirector));
```

Comment:
It looks like this is the last use of `redirector` in this scope so shouldn't it be kj::mv() rather than kj::addRef()?